delivery_carrier_label_roulier/stock.py

```diff
@@ -253,20 +245,19 @@ def _roulier_convert_address(self, partner):
                     # it's a None: nothing to do
                 else:  # it's a boolean: keep the value
                     address[elm] = partner[elm]
-
-        # get_split_adress from partner_helper module
-        res = partner._get_split_address(partner, 3, 38)
-        address['street2'], address['street1'], address['street3'] = res
-
-        # parent_id is None if it's a company
-        if 'parent_id' in address:
-            address['company'] = address['parent_id'].name
-            del address['parent_id']
-
+        if partner.parent_id.is_company:
+            address['company'] = partner.parent_id.name
         # Codet ISO 3166-1-alpha-2 (2 letters code)
         address['country'] = partner.country_id.code
         return address
 
+    def _roulier_clean_phones(self, address):
```

Comment:
I'm not a big fan of this. It should not be the role of this module to ensure consistancy of data.
What if user put "04/78/03[...]" or "04.78.03.[...]" or "+3347803"  ? 

But let's keep it here for the moment. 

I suggest to: 
 - call this method from _roulier_convert_address 
 - return address

What do you think ?